src/base/base.scss

```diff
@@ -77,10 +77,74 @@ button {
 }
 
 details {
-  summary { cursor: pointer; }
+  summary {
+    cursor: pointer;
+  }
 
   &:not([open]) {
     // Set details content hidden by default for browsers that don't do this
-    > *:not(summary) { display: none !important; }
+    > *:not(summary) {
+      display: none !important;
+    }
+  }
+}
+
+// global focus styles
+
+a,
+button,
+[role='button'],
+input[type='radio'],
+input[type='checkbox'] {
+  transition: 80ms cubic-bezier(0.33, 1, 0.68, 1);
+  transition-property: color, background-color, box-shadow, border-color;
+  // fallback :focus state
+  &:focus {
+    @include focusOutline;
+
+    // remove fallback :focus if :focus-visible is supported
+    &:not(:focus-visible) {
+      outline: solid 1px transparent;
+    }
+  }
+
+  // default focus state
+  &:focus-visible {
+    @include focusOutline;
+  }
+}
+
+a:not([class]),
```

Comment:
Should the [`Link`](https://primer-efelvbnsa-primer.vercel.app/css/components/links) component be included here too? The `outline-offset: -2px;` can sometimes make it hard to read the link text.

![image](https://user-images.githubusercontent.com/378023/163924813-7fb48457-7db7-4fb1-ba4e-544944e6abca.png)

Also there are probably some cases where `a:not([class])` would not be effective, but still desired.. e.g. when a link is shown responsively like `<a class="d-none d-md-block">Link</a>`. Hard to fix without adding tons of exceptions for utility classes. 🤷

Reply:
Links are tricky! I changed this to `0px` offset as a happy medium, and added that offset to the PCSS `Link` component. There were a lot of edge cases in dotcom where having a larger offset wasn't desirable, primarily where an additional utility or class was used. I felt like `:not()` was an appropriate safeguard for now.